### DISCUSSION TITLE
Podman Compose

### DISCUSSION THREAD

**Discussion by trymeouteh** [containers/podman#21116] [Ideas]
Create "podman-compose" which will be the same as docker compose but within podman cli itself. No need to install and secondary podman-compose package, podman compose can be apart of podman itself.

**Comment by vrothberg** [comment]
Thanks for reaching out!  Such a command already exists (see https://github.com/containers/podman/blob/main/docs/source/markdown/podman-compose.1.md.in).   You may be using an older version of Podman if you don't see it.

**Reply by trymeouteh** [reply]
I was not aware it was already a thing being on Ubuntu and using the podman package from the official ubuntu repo
